addon/ng2/commands/github-pages-deploy.ts

```diff
@@ -52,8 +53,16 @@ module.exports = Command.extend({
       type: String,
       default: '',
       description: 'Github username'
+    }, {
+      name: 'output-path',
+      type: String,
+      default: 'dist/',
```

Comment:
I added the flag for flexibility, there was a use case where someone wanted to have different environments build to folders, having a flag would allow this. If you still feel it is unnecessary I'm happy to back it out.